extensions/amp-pinterest/0.1/pin-widget.js

```diff
@@ -293,6 +306,8 @@ export class PinWidget {
 
     // listen for clicks
     structure.addEventListener('click', this.handleClick.bind(this));
+    // Handle Space and Enter while selected for a11y
+    structure.addEventListener('keydown', this.handleKeyDown.bind(this));
```

Comment:
Change this to `keypress`. A real button only performs an action when you actually release the key.

If it isn't too much trouble, also consider just using a real button.

Reply:
Changed to keypress. I think I need to take a closer look at the UI per https://github.com/ampproject/amphtml/issues/23116 anyway, so I'll look at the button thing along with the rest of the UI in a following PR.